hydra-main/src/main/java/com/addthis/hydra/job/spawn/balancer/SpawnBalancer.java

```diff
@@ -1403,15 +1403,38 @@ private JobTaskItemByHostMap generateTaskCountByHost(List<HostState> hosts, Iter
         return rv;
     }
 
-    private List<JobTaskMoveAssignment> pushTasksOntoHost(HostState host, Iterable<HostState> heavyHosts) {
+    private List<JobTaskMoveAssignment> pushTasksOntoHost(HostState host, Iterable<HostState> heavyHosts,
```

Comment:
`moveLimit` seems to be taken from the `config` object. You can remove that argument and reference `config` directly since this method is `private` scoped.